libs/akita/src/lib/queryEntity.ts

```diff
@@ -291,22 +291,24 @@ export class QueryEntity<S extends EntityState, EntityType = getEntityType<S>, I
    * Listen for entity actions
    *
    *  @example
-   *
    *  this.query.selectEntityAction(EntityActions.Add);
    *  this.query.selectEntityAction(EntityActions.Update);
    *  this.query.selectEntityAction(EntityActions.Remove);
    *
+   *  this.query.selectEntityAction(EntityActions.Add, EntityActions.Update, EntityActions.Remove)
+   *
    *  this.query.selectEntityAction();
    */
-  selectEntityAction(action: EntityActions): Observable<IDType[]>;
+  selectEntityAction(...actions: EntityActions[]): Observable<{ [Name in EntityActionsNames]?: IDType[] }>;
   selectEntityAction(): Observable<EntityAction<IDType>>;
-  selectEntityAction(action?: EntityActions): Observable<IDType[] | EntityAction<IDType>> {
-    if (isUndefined(action)) {
+  selectEntityAction(...actions: EntityActions[]): Observable<{ [Name in EntityActionsNames]?: IDType[] } | EntityAction<IDType>> {
+    if (isEmpty(actions)) {
       return this.store.selectEntityAction$;
     }
+
     return this.store.selectEntityAction$.pipe(
```

Comment:
This is a breaking change @xbaun

Reply:
The problem (if it is) is how to distinguish between results of different entity actions. On the other hand, we could of course simply return the ids as we do now and not distinguish between them.

Reply:
Another option is to overload it an accept an array instead.

Reply:
I think we were talking about different things: I meant that the value of the subscription is a breaking change. To overload the function for backwards compatibility is the other.

Reply:
Yes, that's what I meant when saying a braking change.

Reply:
Just to make it clear, the function will return `ids` array for single type, and `{ [EntityActions.Type]: ids }` for array input?

Reply:
@talohana now I get it, sounds good 👍

Reply:
Yes, that's what I meant.

Reply:
I've updated accordingly, just not 100% the function return type is the best I can get out of it.